### PR TITLE
DDP-5672 Longitudinal Kit Restrictions

### PR DESCRIPTION
**Pull request by zyme**
Updates from the [original PR](https://github.com/broadinstitute/ddp-study-manager/pull/129) to ensure proper target branch (along with other PR feedback).

### CODE REVIEW

File: src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java [hunk 1]
```diff
@@ -80,10 +72,7 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                         if (kitType != null) {
                                             KitRequestSettings kitRequestSettings = kitRequestSettingsMap.get(kitType.getKitTypeId());
 
-                                            ArrayList<KitRequest> orderKit = kitsToOrder.get(kitRequestSettings);
-                                            if (orderKit == null) {
-                                                orderKit = new ArrayList<>();
-                                            }
+                                            ArrayList<KitRequest> orderKit = new ArrayList<>();
```
Comment (SimoneMaiwald) [inline src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: could be removed as well =)

Comment (zyme) [reply src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: Screen Shot 2021-03-17 at 11 40 39 AM

Comment (zyme) [reply src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: How far should this go?

Comment (zyme) [reply src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: Screen Shot 2021-03-17 at 11 42 45 AM

Comment (zyme) [reply src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: Initially I went a bit further--I think my mistake was also removing the call to `addSubKits()`?

Comment (SimoneMaiwald) [reply src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: yes the call to addSubKits should stay. I normally love ripping things out, so if you want you can leave it and next time I will touch that class I will remove unused things =)

Comment (zyme) [reply src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java]: Done!  One more look?

**Review by SimoneMaiwald — APPROVED** [review]

**Pull request merged**